### DISCUSSION TITLE
Generalize the Devise emailer for invitations

### DISCUSSION THREAD

**Discussion by leewynne** [forem/forem#16200] [Feature Requests]
At the moment Forem is using default values and text for the devise invitation mailer. I think the best solution for this is to configure the text used to send invitations out via SiteConfig.

 I think an invitation needs to be more personal from the originating Forem

**Comment by ellativity** [comment]
Connecting this one to https://github.com/forem/forem/discussions/15972 and forem/rfcs#324 (internal reference)

**Comment by Steven3261** [comment]
How will we use it by confsite. As I am trying to do it for the website of [Stripping services Boston](http://commercialcleaningservicesboston.com/wax-services-boston/) but I am unable. Although I have readed bundles of articles and applied all the methods which they are talking but not finding the final solution for it.
